### ISSUE TITLE
Edit messages?

### ISSUE THREAD

**Issue by luk3yx** [Zippy1/ls_commands#3]
Hello,

Could it make sure messages have capital letters, full stops (or other punctuation), and extend acronyms?

Thanks,

luk3yx

**Comment by Zippy1** [comment]
Sure, I'll fix that
you mean like
from "hi" to "Hi."?

**Comment by luk3yx** [comment]
Yes, I do. Maybe kick players without the 'server' privilege who try and swear.
